### DISCUSSION TITLE
how i can set network interface

### DISCUSSION THREAD

**Discussion by lukasa1993** [mitmproxy/mitmproxy#6255] [Q&A]
so i have wifi and ethernet how i can force proxy to go out with wifi ?

**Comment by mhils** [comment]
This is not configurable (yet), mitmproxy uses whatever is the default interface for you.

**Reply by lukasa1993** [reply]
any plans to add it ?

**Reply by mhils** [reply]
Happy to look at PRs, but it's not very high up on my own priority list. :)

**Reply by lukasa1993** [reply]
okay
